interviews.md

```diff
@@ -64,23 +65,33 @@ title: "Archive of King Gizzard interviews"
 
 * Nov 22: AudioTechnology, [Audio technology King Gizzard’s Lo-Fi Gut Feeling](https://www.audiotechnology.com/features/king-gizzards-lo-fi-gut-feeling)
 * Jun 17: (video) [Live in Limbo](https://www.youtube.com/watch?v=8hW_OeSrLQA) with Stu
+* Mar 2: (video) SRF 3, [King Gizzard & The Lizard Wizard: Australische Reptilien in Zurich gesichtet](https://www.youtube.com/watch?v=nPBDcgNw6Pc) with Stu
```

Comment:
Is this in German? should we label that?

Reply:
It's a German channel but the interview is in English.